packages/suite-desktop-api/src/api.ts

```diff
@@ -69,6 +73,8 @@ export interface InvokeChannels {
     'metadata/write': (options: { file: string; content: string }) => InvokeResult;
     'server/request-address': (route: string) => string | undefined;
     'tor/toggle': (shouldEnableTor: boolean) => InvokeResult;
+    'bridge/toggle': () => InvokeResult;
+    'bridge/get-status': () => InvokeResult;
```

Comment:
If there is a payload, it should be `() => InvokeResult<Status>` instead, no?

Reply:
thanks, fixed in 2b92f68103839b87ef3e5996f8d1a686b2235eda